pkg/generic/jsonthrift_codec.go

```diff
@@ -41,11 +43,24 @@ type jsonThriftCodec struct {
 	provider         DescriptorProvider
 	codec            remote.PayloadCodec
 	binaryWithBase64 bool
+	enableDynamicgo  bool
+	convOpts         conv.Options
 }
 
-func newJsonThriftCodec(p DescriptorProvider, codec remote.PayloadCodec) (*jsonThriftCodec, error) {
+func newJsonThriftCodec(p DescriptorProvider, codec remote.PayloadCodec, opts ...Option) (*jsonThriftCodec, error) {
 	svc := <-p.Provide()
-	c := &jsonThriftCodec{codec: codec, provider: p, binaryWithBase64: true}
+	var c *jsonThriftCodec
+	if opts == nil {
```

Comment:
json generic implementation here should also be changed like http.

Reply:
fixed